docs/deployment/getting_started_locally.md

```diff
@@ -12,6 +12,7 @@ Based on [Skaffold](https://skaffold.dev/), the container images for all require
 
 ## Prerequisites
 
+- Make sure that you have followed the [Local Setup guide](https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md) up until the [Get the sources](https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md#get-the-sources) step.
```

Comment:
According to @dimitar-kostadinov , absolute links pointing to the same repo should be avoided - ref https://github.com/gardener/gardener/pull/5894#discussion_r863525721.

You could use relative links instead.